build/templates/README.md

```diff
@@ -367,6 +367,7 @@ For details see the [`values.yaml`](values.yaml) file.
 | `serviceMonitor.annotations`                              | Additional annotations of ServiceMonitor                        | `{}`                                                  |
 | `serviceMonitor.interval`                                 | ServiceMonitor scrape metrics interval                          | `10s`                                                 |
 | `serviceMonitor.scrapeTimeout`                            | ServiceMonitor scrape timeout                                   | `nil`                                                 |
+| `serviceMonitor.tlsConfig`                                | Additional TLS configuration of ServiceMonitor                  | `{}`                                                  |
```

Comment:
You must add this change in the cockroachdb/README.md and README.md file as well

Reply:
@mprimeaux You missed this one.